api-gateway/api-gw/consul-api-gateway.yaml

```diff
@@ -1,34 +1,10 @@
-apiVersion: api-gateway.consul.hashicorp.com/v1alpha1
-kind: GatewayClassConfig
-metadata:
-  name: test-gateway-class-config
-spec:
-  useHostPorts: true
-  logLevel: trace
-  consul:
-    scheme: https
-    caSecret: consul-ca-cert
-    ports:
-      http: 8501
-      grpc: 8502
----
-apiVersion: gateway.networking.k8s.io/v1alpha2
-kind: GatewayClass
-metadata:
-  name: test-gateway-class
-spec:
-  controllerName: "hashicorp.com/consul-api-gateway-controller"
-  parametersRef:
-    group: api-gateway.consul.hashicorp.com
-    kind: GatewayClassConfig
-    name: test-gateway-class-config
 ---
 apiVersion: gateway.networking.k8s.io/v1alpha2
 kind: Gateway
 metadata:
-  name: test-gateway
+  name: example-gateway
 spec:
-  gatewayClassName: test-gateway-class
+  gatewayClassName: consul-api-gateway
```

Comment:
this is the name of the base gateway class installed by `helm`-based installation